chains/ethereum/events.go

```diff
@@ -9,9 +9,9 @@ import (
 )
 
 func (l *listener) handleErc20DepositedEvent(destId msg.ChainId, nonce msg.Nonce) (msg.Message, error) {
-	l.log.Debug("Handling deposited event", "dest", destId, "nonce", nonce)
+	l.log.Info("Handling fungible event", "dest", destId, "nonce", nonce)
```

Comment:
For these logs I think it may be more clear to have deposited included after the deposit event type